src/base/utils/fluxes.jl

```diff
@@ -32,3 +32,39 @@ function metabolite_fluxes(model::MetabolicModel, flux_dict::Dict{String,Float64
     end
     return consuming, producing
 end
+
+"""
+    atom_fluxes(model::MetabolicModel, flux_dict::Dict{String, Float64})
+
+Return a dictionary mapping the flux of atoms across a flux solution given by `flux_dict`
+using the reactions in `model` to determine the appropriate stoichiometry.
+
+Note, if a model is mass balanced there should be not net flux of any atom. By removing
+reactions from the flux_solution you can investigate how that impacts the mass balances.
+
+# Example
+```
+# Find flux of Carbon through all metabolic reactions except the biomass reaction
+delete!(fluxes, "BIOMASS_Ecoli_core_w_GAM")
+atom_fluxes(model, fluxes)["C"]
+```
+"""
+function atom_fluxes(model::MetabolicModel, flux_dict::Dict{String,Float64})
+    S = stoichiometry(model)
```

Comment:
this is a bit heavyweight, isn't there the `reaction_stoichiometry` accessor for this?

Reply:
yep, simplified the code